_episodes/01-introduction.md

```diff
@@ -6,18 +6,22 @@ questions:
 - "What are containers, and why might they be useful to me?"
 objectives:
 - "Show how software depending on other software leads to configuration management problems."
-- "Identify the problems that software installation problems can pose for research."
-- "Give two examples of how containers can solve software configuration problems." 
+- "Identify the problems that software installation can pose for research."
+- "Explain the advantages virtualization has over traditional computing."
```

Comment:
Ideally, it might be good to be more specific about the type of virtualisation (I'm struggling to think of a good term though: "userspace virtualisation" is too "computer science", "application virtualisation" isn't quite right either). Could the "traditional computing" clause be left out possibly given that we talk elsewhere of how old virtualisation is—it felt like the point could be made on the basis of the "advantages" themselves?

Reply:
What do you think about changing line 10 to...
```suggestion
- "Explain how using containers can solve software configuration problems"
```
this will keep the original language of line 10 but remove the two examples part

Reply:
Oops realize the two examples one is right below, the diff was just confused.  What do you think about this one instead?
```suggestion
- "Explain the advantages of application virtualization."
```